mount/mount_windows.go

```diff
@@ -105,14 +105,11 @@ func (mounter *Mounter) MountSensitive(source string, target string, fstype stri
 			klog.Warningf("SMB Mapping(%s) returned with error(%v), output(%s)", source, err, string(output))
 			if isSMBMappingExist(source) {
 				valid, errPath := isValidPath(source)
-				if errPath != nil {
-					return errPath
-				}
-				if valid {
+				if valid && errPath == nil {
 					err = nil
 					klog.V(2).Infof("SMB Mapping(%s) already exists and is still valid, skip error", source)
 				} else {
-					klog.V(2).Infof("SMB Mapping(%s) already exists while it's not valid, now begin to remove and remount", source)
+					klog.V(2).Infof("SMB Mapping(%s) already exists while it's not valid, return error: %v, now begin to remove and remount", source, errPath)
 					if output, err = removeSMBMapping(source); err != nil {
 						return fmt.Errorf("Remove-SmbGlobalMapping failed: %v, output: %q", err, output)
 					}
```

Comment:
return the error too if the newSMBMapping below failed. So line 120, can log success case.

Reply:
fixed